pkg/controller/registry/resolver/source_csvs.go

```diff
@@ -85,6 +91,20 @@ func (s *csvSource) Snapshot(ctx context.Context) (*cache.Snapshot, error) {
 			continue
 		}
 
+		if cachedSubscription, ok := csvSubscriptions[csv]; !ok || cachedSubscription == nil {
```

Comment:
I think you can just check for existence (`ok`)?

Reply:
belt + suspenders :D